lib/web_ui/dev/test_runner.dart

```diff
@@ -404,13 +460,20 @@ class TestCommand extends Command<bool> with ArgUtils {
           'BUILD_MAX_WORKERS_PER_TASK': io.Platform.environment['BUILD_MAX_WORKERS_PER_TASK'],
       },
     );
-    stopwatch.stop();
-    print('The build took ${stopwatch.elapsedMilliseconds ~/ 1000} seconds.');
 
     if (exitCode != 0) {
       throw ToolException(
           'Failed to compile tests. Compiler exited with exit code $exitCode');
     }
+
+    if (forCanvasKit) {
+      final io.Directory canvasKitTemporaryOutputDirectory = io.Directory(path.join(environment.webUiRootDir.path, canvasKitOutputRelativePath, 'test', 'canvaskit'));
+      final io.Directory canvasKitOutputDirectory = io.Directory(path.join(environment.webUiBuildDir.path, 'test', 'canvaskit'));
```

Comment:
dartfmt.

Line looks very long to me. Since I always use auto format on this file, it will get updated next day with my other change anyway, but it is better to do the formatting in the same PR :)

Reply:
Done (also applied dartfmt to environment.dart)

Reply:
nice! thanks